src/markdown-pages/docs/manage-ecosystem/runs-delete.md

```diff
@@ -0,0 +1,28 @@
+---
+path: "/docs/manage-ecosystem/cli-runs-delete"
+title: "Deleting test run results"
+---
+
+Over time, test run results build up and take up space in the Galasa Ecosystem. You can delete test runs which have completed in the past by using the `galasactl runs delete` command.
+
+You can view the full command syntax in the <a href="https://github.com/galasa-dev/cli/blob/main/docs/generated/galasactl_runs_delete.md" target="_blank">Galasa cli repository</a>.
+
+## Working with the `galasactl runs delete` command
+
+The following example deletes a test run called _C1234_: 
+
+```
+galasactl runs delete --name C1234
```

Comment:
If the `GALASA_BOOTSTRAP` environment variable isn't set then this would need a `--bootstrap` flag as well (this applies to all CLI commands that need to talk to the ecosystem's API server), but it reads a lot nicer without the `--bootstrap` flag!